cmd/crates/soroban-spec-typescript/ts-tests/src/util.ts

```diff
@@ -1,15 +1,22 @@
 import { spawnSync } from "node:child_process";
 import { Address, Keypair } from "@stellar/stellar-sdk";
-import { basicNodeSigner } from "@stellar/stellar-sdk/ContractClient";
+import { basicNodeSigner } from "@stellar/stellar-sdk/contract";
 
-const rootKeypair = Keypair.fromSecret(spawnSync("./soroban", ["keys", "show", "root"], { shell: true, encoding: "utf8" }).stdout.trim());
+const rootKeypair = Keypair.fromSecret(
+  spawnSync("./soroban", ["keys", "show", "root"], {
+    shell: true,
+    encoding: "utf8",
+  }).stdout.trim(),
+);
 
 export const root = {
   keypair: rootKeypair,
   address: Address.fromString(rootKeypair.publicKey()),
-}
+};
 
 export const rpcUrl = process.env.SOROBAN_RPC_URL ?? "http://localhost:8000/";
-export const networkPassphrase = process.env.SOROBAN_NETWORK_PASSPHRASE ?? "Standalone Network ; February 2017";
+export const networkPassphrase =
+  process.env.SOROBAN_NETWORK_PASSPHRASE ??
+  "Standalone Network ; February 2017";
```

Comment:
It actually is. It's exported as enum `Networks` from stellar base. We should probably update this file to use that import, instead of the string.

Reply:
I wouldn't put too much concern on this. The string hasn't changed since 2017, so we don't need to engineer for something that hasn't changed in 7 years.